frontend/packages/dev-console/integration-tests/features/getting-started/getting_started _to_dev_perspective.feature

```diff
@@ -0,0 +1,11 @@
+Feature: Create Sample Application
+    As a user I want to create the Sample Application from Add options
+
+Background:
+    Given user is developer
+
+@regression
+Scenario: Logging to openshift devconsole
```

Comment:
Scenario : Developer perspective display on login to open shift application
Given user is at login page
When user enters the username as "username"
And password as "password"
And clicks login
Then user redirects to developer perspective